src/core/server/saved_objects/service/lib/repository.ts

```diff
@@ -1330,6 +1330,7 @@ export class SavedObjectsRepository {
       id: raw._id,
       index: this.getIndexForType(type),
       refresh: toRefresh(refresh),
+      _source: 'true',
```

Comment:
I hope the new "@elastic/elasticsearch": "7.9.0-rc.2" will have it fixed https://github.com/elastic/elasticsearch/blob/f29fe0643d1c59a6f51dd08155c5a9695caf1c26/rest-api-spec/src/main/resources/rest-api-spec/api/update.json#L60

Reply:
Nope, it will be wrong, unfortunately, because the type is `list`, which as you can see [here](https://github.com/elastic/elasticsearch/tree/f29fe0643d1c59a6f51dd08155c5a9695caf1c26/rest-api-spec#type-definition) means an array of strings.

Reply:
Yes.